hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DFSStripedInputStream.java

```diff
@@ -554,4 +555,19 @@ public synchronized void releaseBuffer(ByteBuffer buffer) {
     throw new UnsupportedOperationException(
         "Not support enhanced byte buffer access.");
   }
+
+  @Override
+  public synchronized void unbuffer() {
+    closeCurrentBlockReaders();
+    if (curStripeBuf != null) {
+      curStripeBuf.clear();
```

Comment:
Probably a minor issue. Won't we be doing this curStripeBuf.clear() step twice in the unbuffer method? The first time is in Line #127-129.

Reply:
@avijayanhwx Thanks for review! Good point! It cleared twice, I removed the clear in the unbuffer. Could you help review again? Thanks!

Reply:
Thanks for the changes!

Reply:
@avijayanhwx Thanks for review again! The parityBuf can be cleared in method getParityBuffer() line # 133 - 140